components/places/src/ffi.rs

```diff
@@ -113,6 +267,38 @@ impl PlacesConnection {
             )
         })
     }
+
+    fn bookmarks_get_tree(&self, item_guid: &Guid) -> Result<Option<PublicNode>> {
+        self.with_conn(|conn| {
+            let root_id = Guid::from(item_guid.as_str());
```

Comment:
This probably isn't necessary - just use `item_guid` directly (and ditto the next function)